python/app/redirect.py

```diff
@@ -101,9 +99,6 @@ def write(self, msg):
 
         If tee, then also write to stdout.
         """
-        if isinstance(msg, unicode):
```

Comment:
This is a case where it would make sense to put a comment in the PR explaning why we are removing code.

Reply:
Good point!
So we are removing this line as it is not compatible with python 3, and this module is designed to run without sgtk being present so we can't rely on the existence of `six` to smooth things over.

I was decided however it was better to add a check to the check to the `OutputStreamWidget.add_output` as this was where it was going to get added to the UI, so we should sanitize it there. It still can't rely on six being there, but it will try to use it if six could be imported.